### PR TITLE
Mirror OBS signing key on owncloud.org

### PR DESCRIPTION
**Pull request by LukasReschke**
I'm absolutely not sure if that is the way to add static files and link to them, so feel free to make adjustments.

@MorrisJobke FYI

### CODE REVIEW

File: install-instructions.php [hunk 1]
```diff
@@ -73,6 +73,7 @@
 	    <div class="col-md-6">
 		<p>You can find further instructions in the <a href="<?php echo $DOCUMENTATION_ADMIN; ?>">ownCloud Admin Manual</a>.</p>
 		<p>Note:</br> Once the setup is done, the ownCloud installer will remove itself automatically. We recommend a subscription on our low-traffic <a href="http://mailman.owncloud.org/mailman/listinfo/announcements">announcement mailing list</a> for notifications on updates and security issues. The public ownCloud <a href="<?php echo $OWNCLOUD_GPG; ?>">GPG key</a>.</p>
+	    <p>The key used to sign our OBS packages can be found here: <a href="https://owncloud.org/assets/files/obs-release-signing.gpg">OBS Release Signing key</a></p>
```
Comment (tomneedham) [inline install-instructions.php]: I'd make this relative, so it works on the staging and local servers.

Comment (LukasReschke) [inline install-instructions.php]: Adjusted

**Comment by LukasReschke** [comment]
@tomneedham Please re-review :)

**Comment by tomneedham** [comment]
Super :+1:

**Pull request merged**